app/konnect/dev-portal/index.md

```diff
@@ -1,13 +1,50 @@
+
+For more information on this topic, we offer several pieces of documentation depending on your use case.
+
+For Kong Konnect Administrators: 
+
+* [Enabling and disabling application registration](/konnect/dev-portal/applications/enable-app-reg/) - This explains how to grant and rescind application registration access. 
+Manage developer access - This doc details to a Kong Konnect administrator all of the different options they have available to them to manage developer access to their Dev Portal. The Dev Portal offers administrators the ability to approve and reject requests for access to the Dev Portal.
+
+* [Manage developer access](/konnect/dev-portal/access-and-approval/manage-devs/)- This doc details to a Kong Konnect administrator all of the different options they have available to them to manage developer access to their Dev Portal. The Dev Portal offers administrators the ability to approve and reject requests for access to the Dev Portal.
+
+
+* [Manage application registration requests](/konnect/dev-portal/access-and-approval/manage-devs/) -  When developers want to create applications on your Dev Portal, they will have to create a request. Requests can be managed from within Konnect. This document contains instructions on how to manage application requests. If you are interested in automatically allowing application requests automatically, read our guide on handling [auto approval](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/). 
+
+For Developers: 
+
+* [Developer registration and account management](/konnect/dev-portal/access-and-approval/dev-reg/) - This explains how to register for a Dev Portal as well as the registration process and different registration states, for developers who want to register for the platform. It also explains account management information. 
+
+* [Manage Applications](/konnect/dev-portal/applications/dev-apps/) - Managing application registration requests.
+
+* [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform. 
+
+## Customization
+
+Konnect has built-in customization options for managing the Dev Portal so that you can build a consistent experience for the consumers of your API. Customizing your Dev Portal to reflect the likeness of your brand can help convince Developers to create applications with your services. With Konnect, you don’t need to have knowledge of any web frameworks, because you customize everything from the application. Konnect also has built-in themes if customization is not your immediate priority. 
 
-## See also
+For more information please read our doc on [customization](/konnect/dev-portal/customization/).
 
 * [Application Overview](/konnect/dev-portal/applications/application-overview)
```

Comment:
@Guaris did you mean to leave this last link in here? Looks like it doesn't belong anymore.